hypervisor/src/hypervisor/config.rs

```diff
@@ -139,6 +139,9 @@ pub struct Partition {
     ///
     /// Use this to expose a path / file / device file from the host environment
     /// to the inside of a partitions.
+    ///
+    /// The first path is the path in the host environment, while the second
+    /// path is a relative one inside of the partition.
```

Comment:
the second path should not be a relative path. because from the perspective of the partition and the partition developer the path is an absolute path from the root of the partition.